### PR TITLE
fix OS name

### PR DESCRIPTION
**Pull request by mairaw**
Anonymous feedback: "There is no such thing as Windows Server 2016 (version 1709); it should be Windows Server, version 1709"

https://docs.microsoft.com/en-us/dotnet/framework/install/on-windows-10

/cc @rpetrusha @vivmishra

### CODE REVIEW

File: docs/framework/install/on-windows-10.md [hunk 1]
```diff
@@ -74,4 +74,4 @@ If you cannot get the correct version of the .NET Framework installed, you can [
 
 [.NET Downloads](https://www.microsoft.com/net/download?utm_source=ms-docs&utm_medium=referral)   
 [Troubleshoot blocked .NET Framework installations and uninstallations](troubleshoot-blocked-installations-and-uninstallations.md)   
-[Install the .NET Framework for developers](guide-for-developers.md)
\ No newline at end of file
+[Install the .NET Framework for developers](guide-for-developers.md)
```
Comment (jzeferino) [inline docs/framework/install/on-windows-10.md]: Dont understand why this was considered a changed, but its ok.

**Review by jzeferino — APPROVED** [review]
LGTM.

File: docs/framework/install/on-windows-10.md [hunk 2]
```diff
@@ -22,7 +22,7 @@ You may have arrived on this page after trying to run an application and seeing
 The .NET Framework 4.7.1 is included with:
 
 * [Windows 10 Fall Creators Update (version 1709)](https://www.microsoft.com/software-download/windows10)
-* [Windows Server 2016 (version 1709)](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709)
+* [Windows Server (version 1709)](https://docs.microsoft.com/windows-server/get-started/get-started-with-1709)
```
Comment (richlander) [inline docs/framework/install/on-windows-10.md]: Looks like it should be "Windows Server, version 1709"

File: docs/framework/install/on-windows-10.md (hunk 1 above)
Comment (mairaw) [reply docs/framework/install/on-windows-10.md]: Sometimes that happens with the last line @jzeferino

**Comment by mairaw** [comment]
Good catch @richlander. I’ve fixed it.

**Review by JRAlexander — APPROVED** [review]
LGTM.

**Pull request merged**